select/harness.ts

```diff
@@ -48,6 +51,18 @@ export class SelectHarness extends Harness<Select> {
     field.click();
   }
 
+  async clickAndWaitForMenu() {
```

Comment:
Could we override `click()` and wait for the menu's events there? I'm not sure when you'd want to click the element and not wait for all the events to settle.